compiler error in develop branch: 'unique_ptr' is not a member of ‘std’

```
In file included from /home/luotao02/Paddle/paddle/framework/attribute.h:24:0,
                 from /home/luotao02/Paddle/paddle/framework/attribute.cc:15:
/home/luotao02/Paddle/paddle/framework/type_defs.h:39:31: error: ‘unique_ptr’ is not a member of ‘std’
     std::function<std::vector<std::unique_ptr<OpDescBind>>(const OpDescBind&)>;
                               ^
/home/luotao02/Paddle/paddle/framework/type_defs.h:39:31: error: ‘unique_ptr’ is not a member of ‘std’
/home/luotao02/Paddle/paddle/framework/type_defs.h:39:47: error: template argument 1 is invalid
     std::function<std::vector<std::unique_ptr<OpDescBind>>(const OpDescBind&)>;
                                               ^
/home/luotao02/Paddle/paddle/framework/type_defs.h:39:47: error: template argument 2 is invalid
/home/luotao02/Paddle/paddle/framework/type_defs.h:39:57: error: template argument 1 is invalid
     std::function<std::vector<std::unique_ptr<OpDescBind>>(const OpDescBind&)>;
```

--- a/paddle/framework/type_defs.h
+++ b/paddle/framework/type_defs.h
@@ -15,6 +15,7 @@
 #pragma once
 #include <functional>
 #include <map>
+#include <memory>
 #include "paddle/platform/variant.h"
 
 namespace paddle {

